python/lsst/dax/distribution/fakeIngestCfgs/fakedb.json

```diff
@@ -0,0 +1,7 @@
+{"database":"fakedb",
```

Comment:
'fake' just means the data is made up, and all of the configuration files are for 'fakedb'. All the configuration files are pretty tightly linked. If something changes in one, there probably need to be changes to most of the configuration files. The other systems want similar information in different formats. If there's a different database to be created, it will need an entire new set of config files and directories. 

I don't really expect to need that different databases, but I expect there will be a kpm50db that will not be suitable for quick testing on a lone pc, and that will have 'kpm50IngestCfgs'.

Reply:
I think if there was a `config/` directory in the repository root, then `config/fakedb/` would be an understandable path. Right now I keep reading it as ingest itself being "fake", somehow. (I also wonder if there's something more specific to say than just "fakedb", since the other databases this generates will also be "fake")

There's not really a python standard for configuration data (as far as I know), so anywhere that's distinct from the python code should be fine.

Reply:
The fake database is going to be visible along side wise and stripe 82 in qserv and it's not really deserving of of a name like synthetic as it overwhelmingly consists of uncorrelated random numbers. It's only good for bulk testing of qserv and I wouldn't want anyone to think it's something interesting, hence 'fake'.